test/fixtures/fixture-sharepointer.js

```diff
@@ -0,0 +1,29 @@
+module.exports = function(){
```

Comment:
Spent 20 mins with this before saying "fuck it". Seems callsArgWith requires you to pass in the object being stubbed, and doesn't just return a stub function.